.github/workflows/podvm.yaml

```diff
@@ -55,12 +55,25 @@ jobs:
         username: ${{ github.repository_owner }}
         password: ${{ secrets.GITHUB_TOKEN }}
 
+    - name: Determine image tags
+      run: |
+        event_name=${{ github.event_name }}
+        img="${{ inputs.registry }}/podvm-${{ matrix.provider }}-${{ matrix.os }}-${{ matrix.arch }}"
+        tags=""
+
+        case "$event_name" in
+        release)
+          tags="${img}:latest,${img}:${{ github.sha }}" ;;
```

Comment:
I wonder if we should also tag release images with the release version as well as the sha and latest?